README.md

```diff
@@ -1,24 +1,189 @@
-# README
+<h3 align="center"><b>SpendSmart</b></h3>
 
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure to record a presentation video presenting the features of the project and The content of the video should include:

![image](https://github.com/yusufyusie/budget-app/assets/83803414/bf6b02ee-9cbf-47bb-8967-1e2112af48f2)